### PR TITLE
1191 PK to UUID; Catalog

### PR DESCRIPTION
**Pull request by wbyoko**

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: efcms-service/storage/fixtures/seed/trial-sessions.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 [
   {
-    "pk": "trial-session",
-    "sk": "959c4338-0fac-42eb-b0eb-d53b8d0195cc",
+    "pk": "959c4338-0fac-42eb-b0eb-d53b8d0195cc",
```
Comment (codyseibert) [inline efcms-service/storage/fixtures/seed/trial-sessions.json]: I'd use "trial-session-959c4338-0fac-42eb-b0eb-d53b8d0195cc" for PK and SK

File: shared/src/persistence/dynamo/trialSessions/getTrialSessions.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
 const { query } = require('../../dynamodbClientService');
 
 exports.getTrialSessions = async ({ applicationContext }) => {
-  return query({
+  return await query({
```
Comment (kkoskelin) [inline shared/src/persistence/dynamo/trialSessions/getTrialSessions.js:4]: In keeping with the Fresh Prince theme...

**Review by kkoskelin — APPROVED** [review]
I'm good with it. Cool after addressing Cody's comment, perhaps?

**Review by codyseibert — APPROVED** [review]

**Pull request merged**